internal/testutil/api.go

```diff
@@ -0,0 +1,34 @@
+package testutil
+
+import (
+	"encoding/json"
+	"github.com/cirruslabs/cirrus-ci-agent/api"
+	"google.golang.org/protobuf/encoding/protojson"
+	"testing"
+)
+
+func TasksToJSON(t *testing.T, tasks []*api.Task) []byte {
+	var unmarshalledTasks []interface{}
+
+	for _, task := range tasks {
+		var unmarshalledTask interface{}
+
+		marshalled, err := protojson.MarshalOptions{Indent: "  "}.Marshal(task)
```

Comment:
I wonder if we need `EmitUnpopulated: true` 🤔

Reply:
I think it will just tremendously increase the verbosity with the little benefit of detecting `.proto` changes, and these in turn would require us to re-generate the fixture each time a field which is not beneficial for the test case is added or removed.